CHANGELOG.md

```diff
@@ -1,5 +1,79 @@
 # CHANGELOG
 
+## Release 2.4.4 (in development)
+
+### Features added
+- [#2561](https://github.com/scality/metalk8s/issues/2561) - install `kubectl`
+on all master nodes and cleanup old "bootstrap" pods after a bootstrap restore
```

Comment:
No do not talk about bootstrap pods cleanup it's only a command in documentation.

Also I'm not sure you want to put the issue here since the issue is really related to restore when the feature is just that kubectl is installed on all master nodes by default